rainbowsaddle/__init__.py

```diff
@@ -12,6 +12,16 @@
 import functools
 import traceback
 
+enable_statsd = True
+stats = None
+
+try:
+    import statsd
+    stats = statsd.StatsClient(prefix='rainbowsaddle')
+except ImportError:
+    print('Statsd import failed')
+    enable_statsd = False
```

Comment:
Instead of an enable/disable boolean and a stats object, just check `if stats is not None:` throughout